decentraland/unity-renderer
# feat: add members list to the nearby channel

## What does this PR change?
Fix #4610 

The list of members of the `nearby` channel was only visible in the voice chat, so we have implemented the same list there.

### Acceptance Criteria
- [x] Nearby channel show the list of nearby members
- [x] The member list is visible by default

## How to test the changes?
1. Go to: https://play.decentraland.org/?explorer-branch=feat/add-member-list-to-the-nearby-channel
2. Open the `nearby` channel.
3. Check that you can see a new members panel opened by default.
4. Check this panel keeps updated with the players you have around (the content of this list should be the same as the list we see in the Voice Chat feature).

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
